src/plugins/i18n.ts

```diff
@@ -154,7 +154,7 @@ const messages = {
         username: 'Username',
         password: 'Password',
         confirm: 'Confirm Password',
-        invitation: 'Invitation Code,optional',
+        invitation: 'Invitation Code, optional',
```

Comment:
```suggestion
        invitation: 'Invitation Code (optional)',
```

Reply:
I'm sorry I don't know much about Vercel's automated deployment on Github. This workflow seems to be running in my fork repository without getting the key. Please tell me if I need to deploy a rendenmc project on my Vercel account and fill in the SSH key and other information of my project. Or do you think there's another workaround?Or something else is going wrong here?Thanks!

Reply:
> I'm sorry I don't know much about Vercel's automated deployment on Github. This workflow seems to be running in my fork repository without getting the key. Please tell me if I need to deploy a rendenmc project on my Vercel account and fill in the SSH key and other information of my project. Or do you think there's another workaround?Or something else is going wrong here?Thanks!

no, please just ignore Vercel's check

Reply:
> > I'm sorry I don't know much about Vercel's automated deployment on Github. This workflow seems to be running in my fork repository without getting the key. Please tell me if I need to deploy a rendenmc project on my Vercel account and fill in the SSH key and other information of my project. Or do you think there's another workaround?Or something else is going wrong here?Thanks!
>
> no, please just ignore Vercel's check

Then, CI is no problem, code style check has passed.

Reply:
> Then, CI is no problem, code style check has passed.

fix please